app/src/main/AndroidManifest.xml

```diff
@@ -50,7 +50,7 @@
             android:name=".ui.activities.EditProfileActivity"
             android:theme="@style/SettingsActivity" />
         <activity
```

Comment:
I do not see the new Activity declared here, is gonna be done in next Pr's @leighdouglas ?

Reply:
Oh good callout, forgot about the manifest 😅 i'll update now 👍🏽